client/app/telecom/telephony/alias/configuration/changeType/telecom-telephony-alias-configuration-changeType.js

```diff
@@ -1,5 +1,5 @@
 angular.module('managerApp').config(($stateProvider) => {
-  $stateProvider.state('telecom.telephony.alias.configuration.changeType', {
+  $stateProvider.state('telecom.telephony.alias.changeType', {
```

Comment:
Isn't `changeType` the child of `configuration` ?

Reply:
No it isn't, I forgot to set the files to the proper place in folder arborescence